ogr/ogrsf_frmts/miramon/ogrmiramondriver.cpp

```diff
@@ -0,0 +1,249 @@
+    poDriver->SetMetadataItem(GDAL_DMD_LONGNAME,
+                              "MiraMon Vectors (.pol, .arc, .pnt)");
+    poDriver->SetMetadataItem(GDAL_DMD_EXTENSION, "pol");
+    poDriver->SetMetadataItem(GDAL_DMD_EXTENSIONS, "pol arc pnt");
+    poDriver->SetMetadataItem(GDAL_DMD_HELPTOPIC,
+                              "drivers/vector/miramon.html");
+    poDriver->SetMetadataItem(GDAL_DCAP_VIRTUALIO, "YES");
+    poDriver->SetMetadataItem(GDAL_DCAP_Z_GEOMETRIES, "YES");
+
+    poDriver->SetMetadataItem(
+        GDAL_DMD_OPENOPTIONLIST,
+        "<OpenOptionList>"
+        "  <Option name='Height' scope='vector' type='string' "
+        "   description='Sets which of the possible heights is chosen: "
+        "   the first, the highest or the lowest one.'>"
+        "    <Value>First</Value>"
+        "    <Value>Lower</Value>"
+        "    <Value>Highest</Value>"
+        "  </Option>"
+        "  <Option name='iMultiRecord' scope='vector' type='string' "
+        "   description='Sets which of the possible records is chosen: "
+        "   0, 1, 2,... or the Last one. Use JSON when a serialized "
+        "   JSON is wanted'>"
+        "    <Value>0</Value>"
+        "    <Value>1,...</Value>"
+        "    <Value>Last</Value>"
+        "    <Value>JSON</Value>"
+        "  </Option>"
+        "  <Option name='OpenMemoryRatio' scope='vector' type='float' "
+        "   description='Ratio used to enhance certain aspects of memory"
+        "   In some memory allocations, a block of 256 bytes is used."
+        "   This parameter can be adjusted to achieve"
+        "   OpenMemoryRatio*256."
+        "   For example, OpenMemoryRatio=2 in powerful computers and"
+        "   OpenMemoryRatio=0.5 in less powerful computers."
+        "   By increasing this parameter, more memory will be required,"
+        "   but there will be fewer read/write operations to the disk.'>"
+        "    <Value>0.5</Value>"
+        "    <Value>1</Value>"
+        "    <Value>2</Value>"
+        "  </Option>"
+        "  <Option name='OpenLanguage' scope='vector' type='string' "
+        "   description='If the layer to be opened is multilingual "
+        "   (in fact the *.rel* file), this parameter sets the language "
+        "   to be read.'>"
+        "    <Value>ENG</Value>"
+        "    <Value>CAT</Value>"
+        "    <Value>SPA</Value>"
+        "  </Option>"
+        "</OpenOptionList>");
+
+    poDriver->SetMetadataItem(
+        GDAL_DS_LAYER_CREATIONOPTIONLIST,
+        "<LayerCreationOptionList>"
+        "  <Option name='Version' type='string' description='Version of the "
+        "file."
+        "V1.1 is a limited 32 bits for FID and for internal offsets. "
+        "V2.0 is the 64 bits version, with pratically no limits for FID nor for"
+        " internal offsets.' "
+        "default='last_version'>"
+        "<Value>V1.1</Value>"
+        "<Value>V2.0</Value>"
+        "<Value>last_version</Value>"
+        "<Value>nullptr</Value>"
```

Comment:
I guess you don't intend a nullptr value, right ??
```suggestion
```

Reply:
Ok